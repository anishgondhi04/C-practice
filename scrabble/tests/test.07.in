7
6
1 1 H hello
1 1 V howdy
1 5 H yahoo
5 1 V oxoxo
1 3 H watwo
3 1 V latch
0

