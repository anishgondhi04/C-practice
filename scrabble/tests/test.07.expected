.......
.hello.
.o.a.x.
.watwo.
.d.c.x.
.yahoo.
.......
